src/stores/security-two-factor-authentication-use.md

```diff
@@ -31,16 +18,16 @@ With the U2F device configured, the user attaches the device to their system whe
 
    - Line up the red box with the QR code to scan with the camera on the mobile device.
 
```

Comment:
in another topic, we refer to a smart phone.   do you want to pick one for consistency?

Reply:
I think smart phone is better because not all mobile devices can be used. A flip phone is mobile, but not very smart (but cool)